packages/@vuepress/markdown/__tests__/plugins/assetsPlugin.spec.ts

```diff
@@ -396,5 +540,126 @@ describe('@vuepress/markdown > plugins > assetsPlugin', () => {
         )
       })
     )
+
+    const source2 = [
```

Comment:
Why not put them inside `source` and `testCases` directly?

Reply:
already comment the reason